### ISSUE TITLE
Configuration parameter `locatorPriority` is not used

### ISSUE THREAD

**Issue by hufftheweevil** [owntracks/android#1360]
* **App build number**: 2.4.10 (20410051)
* **Android version**: 13
* **Device**: Samsung Galaxy S21
* **Installation source**: Google Play

The configuration parameter `locatorPriority` is included in the documentation, and it is configurable through the UI. However, this parameter is not used anywhere in the code, as far as I can tell.

The priority is set [here](https://github.com/owntracks/android/blob/1141b606666c7a27f9225e218c8a339cf1b349a1/project/app/src/main/java/org/owntracks/android/services/BackgroundService.java#L570). Depending on the mode, it is hard-coded to be 1, 2, or 3. But it never uses the user configuration. So what's the point in setting it?

Or maybe I am missing something in the code?

**Comment by growse** [comment]
Yeah, this functionality got deprecated and removed a while back. Should probably update the docs.

**Comment by growse** [comment]
Closed in https://github.com/owntracks/booklet/commit/f1b09bbb092395dd3a36a73aeb8f61162078f395
